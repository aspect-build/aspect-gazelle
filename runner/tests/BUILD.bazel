load("@aspect_orion//:gazelle.bzl", "gazelle_generation_test")
load("@bazel_gazelle//:def.bzl", "gazelle_binary")

# Disable the gazelle extensions
# gazelle:exclude .

gazelle_binary(
    name = "gazelle_bin",
    testonly = True,
    languages = [
        "@aspect_orion//language/js",
        "//git/testing",
    ],
    visibility = ["//visibility:private"],
)

[
    gazelle_generation_test(
        name = "%s_test" % t,
        dir = t,
        gazelle_binary = ":gazelle_bin",
    )
    for t in [t.replace("/WORKSPACE", "") for t in glob(["*/WORKSPACE"])]
]
