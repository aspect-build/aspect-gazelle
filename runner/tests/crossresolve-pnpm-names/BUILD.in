# gazelle:js_npm_package_target_name pkg
