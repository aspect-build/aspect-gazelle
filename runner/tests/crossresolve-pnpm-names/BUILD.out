load("@deps-test//my:rules.bzl", "x_lib")
load("@npm//:defs.bzl", "npm_link_all_packages")

# gazelle:js_npm_package_target_name pkg

npm_link_all_packages(name = "node_modules")

x_lib(
    name = "i",
    deps = [":node_modules/typescript"],
)
